README.md

```diff
@@ -1,2 +1,202 @@
+|-- releases
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+### Rolling back
+
+In order to rollback, you need to set up the deployment and run the rollback workflow. Workflow, will switch the current folder to the previous release.
+
+Before rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+After rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509150741
+|-- releases
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+### Inputs variables
+
+| Variable       | Type          | Example                  | Discription                                                                                                       |
+| ---------------|:-------------:| :------------------------|-------------------------------------------------------------------------------------------------------------------|
+| server_host    | string        | 59.128.16.24             | The address of the server for deploy                                                                              |
+| environment    | string        | production               | The environment for deploy i.e development, staging, production                                                   |
+| playbook_path  | string        | ci/ansible/playbook.yaml | The path to the file relative to the repository from which the deployment is made, used in Ansistrano deploy task |
+| deploy_path    | string        | ~/deploy/                | The path to deploy on the remote server                                                                           |
+| deploy_repo    | string        | git@github.com:saritasa-nest/ceai-wordpress.git | The path to deploy on the remote server                                                    |
+| deploy_branch  | string        | develop                  | The branch from which the deployment is made                                                                      |
+| python_version | number        | 3.11.4 | Each python version supports a certain range of ansible versions, so we need to specify the current version of python in order to install the latest version of ansible. |
+| runner         | string        | saritasa-rocks-eks       | The runner on which workflow is running: ubuntu, macos, windows, etc                                              |
+
+### Inputs secrets
+
+| Variable        | Discription                                                    |
+| ----------------|:---------------------------------------------------------------|
+| ssh_private_key | The private key for connect to server via ssh                  |
```

Comment:
this has changed in recent changes. pls update

Reply:
Didn't understand what has changed and what do I need to do?

Reply:
@udaltsovra 
attributes changes. You have renamed some of the attrs, yet here in the documentation you're using old naming.

Reply:
fixed